mayastor-test/e2e/common/util.go

```diff
@@ -74,6 +75,9 @@ func GetMSV(uuid string) *MayastorVolStatus {
 		return nil
 	}
 	msVol := MayastorVolStatus{}
+
+	msVol.Replicas = make([]string, 0, 4)
+
 	v := reflect.ValueOf(status)
 	if v.Kind() == reflect.Map {
```

Comment:
I'm not sure if this is expected, but it feels that if for any reason (unless impossible, and if it is impossible I don't understand the if) we return `msVol` filled with default values if `v.Kind != reflect.Map`. And when calling GetMSV we test it for `!= nil` and that test will succeed.

I know this isn't part of the CR, but I'd like to understand if it's a bug.

Reply:
Yes I think it is, we might hit it if the MSV changes again. I'll get it to return nil if v is not of the right type.